opal/corelib/string.rb

```diff
@@ -1575,7 +1575,7 @@ def upto(stop, excl = false, &block)
     return enum_for :upto, stop, excl unless block_given?
     stop = Opal.coerce_to(stop, String, :to_str)
     %x{
-      var a, b, s = self.toString();
+      var a, b, s = self.toString(), value;
```

Comment:
Thanks for your help, @eregon! You can follow this issue if you're interested: https://github.com/opal/opal/issues/1247#issuecomment-164503043